packages/evo/src/__tests__/llm.spec.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  OpenAI,
+  Chat,
+  ConsoleLogger,
+  Logger,
+  InMemoryWorkspace,
+  Env,
+  ChatRole,
+  LlmResponse
+} from "@evo-ninja/agent-utils";
+import dotenv from "dotenv";
+import cl100k_base from "gpt-tokenizer/cjs/encoding/cl100k_base";
+import path from "path";
+import { agentFunctions } from "../agent-functions";
+
+dotenv.config({
+  path: path.join(__dirname, "../../../../.env")
+});
+
+describe('AI Agent Test Suite', () => {
+  const oneMinute = 300 * 1000;
```

Comment:
iiuc this is the timeout that is passed as argument to the test function? if that's the case then i'd say we should name it to `ONE_MINUTE_TIMEOUT` or something like that (or maybe i misunderstood the function of that `oneMinute` var :yum:)